lib/Slic3r/GUI/Plater.pm

```diff
@@ -2432,6 +2432,11 @@ sub select_view {
     }
 }
 
+sub zoom{
+    my ($self, $direction) = @_;
+    $self->{canvas3D}->zoom($direction);
```

Comment:
This will only zoom the 3D plater, and not the other tabs (2D plater, Preview, Toolpaths). It should zoom the currently active tab and leave the others untouched.